scriptworker/task.py

```diff
@@ -39,6 +39,58 @@ def worst_level(level1, level2):
     return level1 if level1 > level2 else level2
 
 
+# get_task_id {{{1
+def get_task_id(claim_task):
+    """Given a claim_task json dict, return the taskId.
+
+    Args:
+        claim_task (dict): the claim_task dict.
```

Comment:
Possibly.  `claim_task` has taskId info and is a superset of `task`.  `task` has the task definition.  If we want all the same arg, we can default to chain or link or context because those have `claim_task` and `task` as properties, or in their context property's properties.  I debated this for a minute, then went with the simplest arg and a docstring.